style.css

```
@@ -596,6 +596,12 @@ footer {
 
 /* extra networks UI */
 
+.extra-network-cards{

```

```suggestion
#txt2img_extra_tabs > .tabitem, #img2img_extra_tabs > .tabitem{
```
This would include the subfolders.